srcpkgs/python3-Pyro4/template

```diff
@@ -1,20 +1,31 @@
 # Template file for 'python3-Pyro4'
 pkgname=python3-Pyro4
-version=4.80
-revision=4
+version=4.81
+revision=1
 wrksrc="${pkgname#*-}-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-serpent"
+# checkdepends="python3-serpent python3-msgpack python3-dill python3-cloudpickle"
+checkdepends="python3-serpent"
 short_desc="Distributed Object Middleware (IPC/RPC) for Python3"
 maintainer="pulux <pulux@pf4sh.de>"
 license="MIT"
 homepage="https://pythonhosted.org/Pyro4/"
```

Comment:
Homepage is dead, please replace with https://pyro4.readthedocs.io/

Reply:
will do